### ISSUE TITLE
Last update information

### ISSUE THREAD

**Issue by dbuenzli** [art-w/sherlocode#10]
It would be nice to have the information on when the index was last updated on the landing page.

**Comment by dbuenzli** [comment]
Ping. Would be nice to have :–) 

I'm trying to use sherlocode to evaluate the impact of breaking changes. But it seems I sometimes fall on outdated sources.

**Comment by OlivierNicole** [comment]
I agree with Daniel. Not knowing when the index was last updated—especially if it could have been years ago—inevitably breeds doubt when using Sherlodoc.
